api/doris/v1/types.go

```diff
@@ -174,6 +174,9 @@ type BaseSpec struct {
 	HostAliases []corev1.HostAlias `json:"hostAliases,omitempty"`
 
 	PersistentVolumes []PersistentVolume `json:"persistentVolumes,omitempty"`
+
+	//SystemParameters for be and cn setting system parameters.
+	SystemParameters []string `json:"systemParameters,omitempty"`
```

Comment:
I think the name `SystemInitialization` is more suitable for the field.
type SystemInitialization struct {
Command []string,
Args[]string,
}